src/App.js

```diff
@@ -98,18 +65,19 @@ function App() {
         </BottomNavigation > 
       </Paper>  
       <Heading />
+      {notes.length ? '' : <h1 style={{marginLeft: 200}}>No Notes To Show</h1>}
```

Comment:
we should not render an empty string if we have notes, if we have notes we should render notes (notes.map...etc)